packages/core/src/App/app.js

```diff
@@ -61,7 +61,6 @@ const initStore = notification_messages => {
     root_store.client.init();
     root_store.common.init();
     root_store.pushwoosh.init();
-    root_store.notifications.init();
```

Comment:
Just for clarification, why are we removing the init method for store?

Reply:
@yashimwong previously, i used `init` to call `setClientNotifications` separately once all other stores initialize and later again when some observables used inside `setClientNotifications` update. but now I figured out that `setClientNotifications` also uses `localize` inside itself which returns the correct translation only when `setClientNotifications` get called. And anytime the language changes we have to not only `handleClientNotifications` in order to show the correct ones but also call `setClientNotifications` before that in order to get a relevant language applied (to call `localize` inside it). init was called before the reaction got triggered. If for example, i leave the call of `setClientNotifications` separately inside `init` and if I add the `current_language` dependency inside the reaction next to `is_uk`, `has_malta_account` & `can_have_mlt_account`, in this case the language gets updated only when we change the language but not when the page is first loaded and `account_settings, account_status`, etc. are updated triggering `handleClientNotifications` call in reaction within the constructor. So the `init` function is redundant now because we need to Set and Handle notifications whenever the main account_settings, account_status, etc. Normally, they update together with the `current_language, is_uk`, `has_malta_account` & `can_have_mlt_account`, that's why we don't need to specify `is_uk`, `has_malta_account` & `can_have_mlt_account` anymore as dependencies and that's why this implementation works. And also this implementation doesn't add extra calls of these functions. They will only be called when their dependencies get actually updated. 🙂

Reply:
I see, awesome explanation 👍🏼  LGTM!